holdings_maintenance/pds4/pds4infoshelf.py

```
@@ -0,0 +1,925 @@
+
+            finally:
+                _ = logger.close()
+
+    except (Exception, KeyboardInterrupt) as e:
+        logger.exception(e)
+        print(sys.exc_info()[2])
+        status = 1
+        raise
+
+    finally:
+        (fatal, errors, warnings, tests) = logger.close()
```

Ruff (0.15.2)

[warning] Unpacked variable `warnings` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)